wizard/WizardLanguage.qml

```diff
@@ -0,0 +1,253 @@
+                anchors.left: parent.left
+                anchors.top: parent.top
+                anchors.leftMargin: 117 * scaleRatio
+                anchors.topMargin: 71 * scaleRatio
+                width: 36 * scaleRatio
+                height: 40 * scaleRatio
+                color: "transparent"
+
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: {
+                        anim.stop();
+                        globe.animFrom = globe.rotation;
+                        globe.animTo = globe.animFrom + 360;
+                        anim.duration = globe.animSlow ? globe.animSpeedNormal : globe.animSpeedSlow;
+                        globe.animSlow = !globe.animSlow;
+                        anim.start();
+                    }
+                }
+            }
+
+            Behavior on opacity {
+                NumberAnimation {
+                    duration: 450;
+                    easing.type: Easing.InCubic;
+                }
+            }
+
+            RotationAnimation on rotation {
+                id: anim
+                loops: Animation.Infinite
+                from: globe.animFrom
+                to: globe.animTo
+                duration: globe.animSpeedNormal
+            }
+        }
+
+        GridLayout {
+            id: buttonsGrid
+            opacity: 0
+            columns: isMobile ? 1 : 2
+            anchors.horizontalCenter: parent.horizontalCenter
+            Layout.topMargin: 20 * scaleRatio
+            Layout.fillWidth: true
+            columnSpacing: 20 * scaleRatio
+
+            MoneroComponents.StandardButton {
+                id: idChangeLang
+                Layout.minimumWidth: 150 * scaleRatio
+                text: "Language"
+
+                onClicked: {
+                    languageSidebar.open();
+                }
+            }
+
+            MoneroComponents.StandardButton {
+                id: btnContinue
+                Layout.minimumWidth: 150 * scaleRatio
+                text: "Continue"
+
+                onClicked: {
+                    if(wizardController.skipModeSelection){
+                        wizardStateView.state = "wizardHome"
+                    } else {
+                        wizardStateView.state = "wizardModeSelection"
+                    }
+                }
+            }
+
+            Behavior on opacity {
+                NumberAnimation {
+                    duration: 350;
+                    easing.type: Easing.InCubic;
+                }
+            }
+        }
+
+        Text {
+            id: versionText
+            opacity: 0
+            anchors.horizontalCenter: parent.horizontalCenter
+            font.bold: true
+            font.pixelSize: 12 * scaleRatio
+            font.family: MoneroComponents.Style.fontRegular.name
+            color: MoneroComponents.Style.defaultFontColor
+            text: Version.GUI_VERSION + " (Qt " + qtRuntimeVersion + ")"
```

Comment:
Perhaps some pre-text. Otherwise this text looks odd without context.
```suggestion
            text: "Wallet version: " + Version.GUI_VERSION + " (Qt " + qtRuntimeVersion + ")"
```

Reply:
That's not odd. That's just how software usually display their versions in splashes and/or intros.